app/graph/mutations/explainer_item_mutations.rb

```diff
@@ -7,5 +7,18 @@ class Create < Mutations::CreateMutation
     argument :project_media_id, GraphQL::Types::Int, required: true
   end
 
+  class SendExplainersToPreviousRequests < Mutations::BaseMutation
+    argument :id, GraphQL::Types::Int, required: true
+    argument :range, GraphQL::Types::Int, required: true
+
+    field :success, GraphQL::Types::Boolean, null: true
+
+    def resolve(id:, range:)
+      explainer_item = ExplainerItem.find_if_can(id, context[:ability])
+      explainer_item.send_explainers_to_previous_requests(range)
+      { success: true }
```

Comment:
Will it always work? You could wrap this in a `begin` ... `rescue` block and return `{ success: false }` in the `rescue` block.

Reply:
Done